src/applications/personalization/profile360/components/Address.jsx

```diff
@@ -28,39 +27,13 @@ class Address extends React.Component {
     // accepted by ErrorableSelect,
     // e.g., from this: `IL: 'Illinois'`
     // to this: `{ value: 'Illinois', label: 'IL' }`
-    let statesList = [];
-
-    // If the city is a military city, just add the military statesList to the list
-    if (this.props.address.city && this.isMilitaryCity(this.props.address.city)) {
-      statesList = militaryStateNames;
-    } else {
-      // Add statesList to list in the correct format
-      _.mapKeys(STATE_CODE_TO_NAME, (value, key) => {
-        statesList.push({ label: value, value: key });
-      });
-      // Add military statesList to full state list
-      militaryStateNames.forEach((militaryState) => {
-        statesList.push(militaryState);
-      });
-      // Alphabetize the list
-      statesList.sort((a, b) => {
-        if (a.label < b.label) {
-          return -1;
-        }
-        if (a.label > b.label) {
-          return 1;
-        }
-        return 0;
-      });
-    }
+    const statesList = [];
 
-    return statesList;
-  }
+    _.mapKeys(STATE_CODE_TO_NAME, (value, key) => {
+      statesList.push({ label: value, value: key });
+    });
 
```

Comment:
Would it possibly make sense to rename "key" to "stateAbbreviation" and "value" to "stateName"? I had to lookup what `STATE_CODE_TO_NAME` looked like to be sure.

Could also initialize `stateList` to `_.mapKeys` by returning `{ label: value, value: key }` instead of doing a `.push`.